### ISSUE TITLE
Enabling SSIL causes error spam in scenes with SubViewport when SubViewport.size is too small

### ISSUE THREAD

**Issue by Antycymon** [godotengine/godot#86631]
### Tested versions

Reproducible in 4.2.1 stable

### System information

Godot v4.2.1.stable - Windows 10.0.19045 - Vulkan (Forward+) - dedicated NVIDIA GeForce GTX 1650 SUPER (NVIDIA; 31.0.15.4617) - AMD Ryzen 5 1600 Six-Core Processor (12 Threads)

### Issue description

Under certain conditions the editor starts spamming error messages:

```
Too many mipmaps requested for texture format and dimensions (6), maximum allowed: (5).
  drivers/vulkan/rendering_device_vulkan.cpp:8627 - Attempted to name invalid ID: 0
  servers/rendering/renderer_rd/storage_rd/render_scene_buffers_rd.cpp:75 - Condition "p_named_texture.texture.is_null()" is true.
  drivers/vulkan/rendering_device_vulkan.cpp:3317 - Parameter "src_tex" is null.
  servers/rendering/renderer_rd/storage_rd/render_scene_buffers_rd.cpp:419 - Condition "named_texture.texture.is_null()" is true. Returning: RID()
  Texture (binding: 0, index 0) is not a valid texture.
  ./servers/rendering/renderer_rd/uniform_set_cache_rd.h:129 - Condition "rid.is_null()" is true. Returning: rid
  drivers/vulkan/rendering_device_vulkan.cpp:7970 - Parameter "uniform_set" is null.
  Uniforms were never supplied for set (3) at the time of drawing, which are required by the pipeline
  Uniforms were never supplied for set (3) at the time of drawing, which are required by the pipeline
  Uniforms were never supplied for set (3) at the time of drawing, which are required by the pipeline
  Uniforms were never supplied for set (3) at the time of drawing, which are required by the pipeline
  servers/rendering/renderer_rd/storage_rd/render_scene_buffers_rd.cpp:419 - Condition "named_texture.texture.is_null()" is true. Returning: RID()
  Image (binding: 0) should provide one ID referencing a texture (IDs provided: 0).
```

Conditions seem to be:
   WorldEnvironment is present and SSIL is enabled
   SubViewport (with Camera attached) is present and **both** x and y values of Subviewport.size property are smaller than 32 px (so for example 1x32 px will work fine, 1x31 will not).

### Steps to reproduce

Create new project and build this hierarchy:

```
Node3D
    ┠╴WorldEnvironment
    ┖╴SubViewport
       ┖╴Camera3D
```

Add default environment, enable SSIL. Go to SubViewport, set both x and y values of size property to be smaller than 32 px. Errors should start to appear even without running the project.

### Minimal reproduction project (MRP)

[SSILbug.zip](https://github.com/godotengine/godot/files/13797765/SSILbug.zip)

**Comment by Calinou** [comment]
This is likely because viewport mipmaps are generated, but the operation fails if the viewport size is too low (presumably because the engine always tries to generate 5 mipmaps: 1×1, 2×2, 4×4, 8×8, 16×16).

**Comment by tprobinson** [comment]
I've encountered this same error spam when SSAO is the only enabled option. GridContainer and FlowContainer seem to handle the SubViewportContainer strangely, so it was getting crushed to 1x1 without me realizing and causing debug build to hang.

**Comment by Jsewill** [comment]
On older hardware, I have encountered the same sort of errors if SSIL is enabled when the project is first opened. No errors if enabled after the project has been opened.

**Comment by AlexandrDzyuba** [comment]
After moving from 4.4.1 stable to 4.5 i get that error but in my case it cause crush.
While game running from engine no error occur, but when project exported to Windows platform game crushes with 
"Too many mipmaps requested for texture format and dimensions (6), maximum allowed: (5)."
And error disappear with disabled SSAO. But if i load location firstly and then enable ssao no crush happen.
 
Godot Engine v4.5.stable.official.876b29033
Vulkan 1.4.303 - Forward+ - Using Device #0: NVIDIA - NVIDIA GeForce GTX 1660 Ti

**Comment by AlexandrDzyuba** [comment]
Yeah i test it more and SSAO || SISIL cause crush, but if i delay enabling for 0.15s then everything fine.
